package.json

```diff
@@ -18,11 +18,6 @@
     "format": "node src format",
     "validate": "node src validate"
   },
-  "husky": {
-    "hooks": {
```

Comment:
Let's keep this. Otherwise you have to have run the build for the hooks to work.